lib/structures/User.js

```diff
@@ -97,6 +109,15 @@ class User extends Base {
         return this.avatar ? this._client._formatImage(Endpoints.USER_AVATAR(this.id, this.avatar), format, size) : this.defaultAvatarURL;
     }
 
+    /**
+    * Get the user's banner with the given format and size
+    * @arg {String} [format] The filetype of the banner ("jpg", "jpeg", "png", "gif", or "webp")
+    * @arg {Number} [size] The size of the banner (any power of two between 16 and 4096)
+    */
+    dynamicBannerURL(format, size) {
```

Comment:
Return type required

Reply:
The return type is missing on all of the other dynamic URL functions, so I figured it wasn't needed. I'll go through and add it to all the other ones with this commit if you think it's necessary.